src/SwiftUI/SwiftUI/Views/Content.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	[SwiftImport (SwiftUILib.Path)]
+	public unsafe record Content : View
```

Comment:
If you look at the `swiftinterface` for SwiftUI (e.g. on my machine, `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/SwiftUI.framework/Modules/SwiftUI.swiftmodule/arm64.swiftinterface`), you'll see that `Content` is actually a type alias for `_ViewModifier_Content<Self>` (where `Self` is the type deriving from `ViewModifier`)